src/authn/auth0.js

```diff
@@ -104,6 +104,10 @@ class Auth0Login {
         throw new Error('unrecognized identity');
       }
 
+      if (profile.app_metadata && profile.app_metadata.groups) {
```

Comment:
I think we want to use `profile.groups`.  The `profile.app_metadata` property isn't in [the schema](https://github.com/mozilla-iam/cis/blob/master/cis/plugins/validation/schema.json)..

Reply:
Worth noting that this file is only used for the legacy UI, and once your Treeherder patch lands, I'll be deleting it :)